.size-limit.js

```diff
@@ -7,18 +7,18 @@ module.exports = [
     name: 'Core - CDN',
     path: 'dist/rudder-analytics.min.js',
     gzip: true,
-    limit: '38 kB',
```

Comment:
This comment is left here as a reminder to revert this change.

Reply:
Addressed

Reply:
What was the root cause for tests contributing to the SDK size?

Reply:
Some issue was there, now it is not affecting the size.